robot-name/damoreno/robot_name.go

```diff
@@ -0,0 +1,40 @@
+package robotname
+
+import "fmt"
+
+type Robot struct {
+	Id string
+}
+
+var availableRobotNames = func() map[string]struct{} {
+	names := make(map[string]struct{})
+
+	for firstLetter := 'A'; firstLetter <= 'Z'; firstLetter++ {
```

Comment:
As I commented on another MR, the exercise says that the names must be random "The names must be random: they should not follow a predictable sequence", while what you do here is pre-generating the names in certain order and you also serve them in that order.